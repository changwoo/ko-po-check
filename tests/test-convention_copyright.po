# -*-text-*-
# test 4 - copyright notice를 번역한 경우
#
msgid ""
msgstr ""
"Project-Id-Version: test 4\n"
"POT-Creation-Date: 2002-07-26 03:40+0900\n"
"PO-Revision-Date: 2002-07-26 03:40+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# E: Copyright notice 번역
#: test.c:4
msgid "Copyright (C) Free Software Foundation, Inc."
msgstr "Copyright (C) 자유소프트웨어 재단."

# E: Copyright notice 번역
#: test.c:4
msgid "Copyright (C) RedHat Software, Inc."
msgstr "저작권 (C) RedHat Software, Inc."

# E: Copyright notice 번역
#: test.c:4
msgid "(C) International Business Machine"
msgstr "(C) 이미 버린 몸"

# E: Copyright notice 번역
#: test.c:4
msgid "Copyright (C) Changwoo Ryu"
msgstr "Copyright (C) Changwoo Ryu"
