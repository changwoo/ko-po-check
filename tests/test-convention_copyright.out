-:18: copyright notice는 번역하면 안 됩니다
-:23: copyright notice는 번역하면 안 됩니다
-:28: copyright notice는 번역하면 안 됩니다
